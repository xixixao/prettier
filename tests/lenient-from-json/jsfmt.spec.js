run_spec(__dirname, ["json"], {
  bracketSpacing: false,
  lenient: true,
  semi: false
});
