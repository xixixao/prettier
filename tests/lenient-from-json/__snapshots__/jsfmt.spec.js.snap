// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`basic.json - json-verify 1`] = `
{
  "a": 1,
  "b": 2
}
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
{
  a: 1
  b: 2
}

`;
